mindsdb/integrations/utilities/time_series_utils.py

```diff
@@ -1,15 +1,19 @@
 import pandas as pd
 from sklearn.metrics import r2_score
+from hierarchicalforecast.core import HierarchicalReconciliation
+from hierarchicalforecast.methods import BottomUp
+from hierarchicalforecast.utils import aggregate
 
 DEFAULT_FREQUENCY = "D"
+DEFAULT_RECONCILER = BottomUp
 
 
 def transform_to_nixtla_df(df, settings_dict, exog_vars=[]):
     """Transform dataframes into the specific format required by StatsForecast.
 
     Nixtla packages require dataframes to have the following columns:
         unique_id -> the grouping column. If multiple groups are specified then
-        we join them into one name using a | char.
+        we join them into one name using a / char.
```

Comment:
Any reason in particular for this change?

EDIT: OK I see why, much better to represent hierarchies with this separator in a single string.